server_manager/www/views/server_view/server_allowance_usage/index.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2024 The Outline Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {css, html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+
+@customElement('server-allowance-usage')
+export class ServerAllowanceUsage extends LitElement {
+  @property({type: String}) message: string;
+  @property({type: Number}) allowanceUsed: number;
+  @property({type: Number}) allowanceLimit: number;
+  @property({type: String}) allowanceUnit: 'gigabyte' | 'terabyte' = 'terabyte';
```

Comment:
*nit* Shall we be "smart" to determine the unit automatically? For example, just let the caller pass a raw bytes: `1,234,567,000`, then we automatically display it as `1.23 GB`?